spec/models/time_profile_spec.rb

```diff
@@ -264,13 +264,13 @@
     end
 
     it "returns unique entries" do
-      tp1a = FactoryBot.create(:time_profile_with_rollup, :id => id_in_region(5, 1))
-      tp1b = FactoryBot.create(:time_profile_with_rollup, :id => id_in_region(5, 2))
-      FactoryBot.create(:time_profile_with_rollup, :days => [1, 2], :id => id_in_region(5, 3))
-      FactoryBot.create(:time_profile, :rollup_daily_metrics => false, :id => id_in_region(5, 4))
-      tp2 = FactoryBot.create(:time_profile_with_rollup, :id => id_in_region(6, 1))
-      FactoryBot.create(:time_profile_with_rollup, :days => [1, 2], :id => id_in_region(6, 2))
-      FactoryBot.create(:time_profile, :rollup_daily_metrics => false, :id => id_in_region(6, 3))
+      tp1a = FactoryBot.create(:time_profile_with_rollup, :id => id_in_region(1, 5))
```

Comment:
is hardocing the rgion number a problem here?  Wonder if it should be current + 1 and current + 2

Reply:
Since it specifies the region for every record created, this should not cause a problem BUT
I will make that change